### PR TITLE
Fixes #34835 - Open Cockpit console in a new tab

### PR DESCRIPTION
**Pull request by ofedoren**
I could probably use `<Link>` from PF4, but just to be consistent with the core I've used this way.

### CODE REVIEW

File: webpack/react_app/components/HostKebab/KebabItems.js [hunk 1]
```diff
@@ -13,7 +13,11 @@ const HostKebabItems = () => {
 
   if (!consoleUrl) return null;
   return (
-    <DropdownItem icon={<CodeIcon />} href={consoleUrl}>
+    <DropdownItem
```
Comment (adamruzicka) [inline webpack/react_app/components/HostKebab/KebabItems.js:16]: Can't we just add `target="_blank"`? It should get passed to whatever the underlying element it

**Review by adamruzicka — COMMENTED** [review]
This works, but isn't it an overkill?

**Comment by adamruzicka** [comment]
@MariaAga as far as I can tell the js test failures shouldn't be related to the changes proposed here. Any idea what that's about?

File: webpack/react_app/components/HostKebab/KebabItems.js (hunk 1 above)
Comment (MariaAga) [reply webpack/react_app/components/HostKebab/KebabItems.js:16]: I agree with this, its more recommended to use
``` js
 <DropdownItem
      icon={<CodeIcon />}
      href={consoleUrl}
      target="_blank"
      rel="noreferrer"
    >
 ```

**Comment by MariaAga** [comment]
Tests failures are not related and I cant really reproduce it. might be a versions thing and I'll see the tests breaking later

**Comment by ofedoren** [comment]
Thanks, @adamruzicka and @MariaAga, looks clean now :)

**Review by adamruzicka — APPROVED** [review]
:shipit:

**Comment by adamruzicka** [comment]
Thank you @ofedoren !

**Pull request merged**